config/config.py

```diff
@@ -0,0 +1,22 @@
+import configparser
+import os
+import shutil
+import json
+
+manifest = json.load(open('config/manifest.json'))
+
+APP_DATA_DIR = '/mrs_app_data'
+CONFIG_DIR = os.path.dirname(os.path.realpath(__file__))  # same dir as this file
+VERSION = manifest['model_version']
+
+if os.path.exists(APP_DATA_DIR):
+    shutil.rmtree(APP_DATA_DIR)
+os.makedirs(APP_DATA_DIR, exist_ok=True)
+
+# Create configparser reader objects
+SETTINGS = configparser.ConfigParser()
+
+# This makes a list of config files for both server and local files
+config_files = list(os.path.join(CONFIG_DIR, x) for x in os.listdir(CONFIG_DIR) if x.endswith('cfg'))
```

Comment:
How are config files read in AIDE?

Reply:
Similar to above, the dicomserver config lives entirely within the container where the app runs. There is no configuration that extends beyond the container, other than the few bits in the manifest file.